src/app/chats/[chatId]/tarot-reading/[resultId]/page.tsx

```diff
@@ -1,11 +1,23 @@
+"use client";
+
 import HeaderContent from "@/shared/components/HeaderContent";
 import MainContent from "@/shared/components/MainContent";
-
+import TarotInteraction from "./components/TarotInteraction";
+import TarotResult from "./components/TarotResult";
+import { useState } from "react";
 export default function TarotReadingResultPage() {
+  const [tarotInteractation, setTarotInteractation] = useState(true);
+  //test
   return (
     <>
       <HeaderContent>{null}</HeaderContent>
-      <MainContent>{null}</MainContent>
+      <MainContent>
+        {tarotInteractation ? (
+          <TarotInteraction setTarotInteractation={setTarotInteractation} />
```

Comment:
P3. 저는 setState를 자식 컴포넌트에게 직접 전달하는 방식을 선호하지 않는 편인 것 같아요. 자식 컴포넌트에게 상태 제어권을 넘기면 업데이트 시점을 예측하기 어려워지는 문제가 있어서요. 개인적인 선호이니 참고만 해주세요. 
추후에 구두로 논의해보아도 재밌는 주제가 될 것 같네요!

단순 예시입니다.
```suggestion
          <TarotInteraction onAnimationComplete={() => setTarotInteractation(false)} />
```